Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM nvidia/cuda:7.5-cudnn4-devel
+FROM nvidia/cuda:7.5-cudnn5-devel
```

Comment:
@dosht Have you tested if TensorFlow installed from the package (.whl) working with CuDNN 5? It's stated in TF docs that:

> The GPU version (Linux only) works best with Cuda Toolkit 7.5 and cuDNN v4. other versions are supported (Cuda toolkit >= 7.0 and cuDNN 6.5(v2), 7.0(v3), v5) only when installing from sources.

(see https://www.tensorflow.org/versions/r0.9/get_started/os_setup.html#requirements).